symengine/__init__.py

```diff
@@ -60,7 +60,6 @@ def lambdify(args, exprs, **kwargs):
 
 
 # To not expose internals
-del lib.symengine_wrapper
```

Comment:
Is this needed?

Reply:
On my system it is needed, since `lib.symengine_wrapper` is not imported anywhere. I can replace it with 
```
try:
   del lib.symengine_wrapper
except NameError:
   pass
```

Reply:
With the latest change, can you try removing this?

Reply:
@isuruf Back to the old behaviour. Works locally, CI still running.

I understand the desire to hide the internals of symengine in the python interface, but removing the `lib.xxx` is a bit drastic I think. A better way might be to rename to something that makes it clear it is internal, e.g. `internal_lib.xxxx` and let the responsibility to use it or not be with the user.

Reply:
The user can still do `import symengine.lib.symengine_wrapper`. They have to explicitly do that instead of having access to it when they do `import symengine`.